src/components/Form/Field/Field.jsx

```diff
@@ -1,9 +1,27 @@
 import React from 'react'
 import { i18n } from '../../../config'
-import ValidationElement from '../ValidationElement'
+import ValidationElement, { newGuid } from '../ValidationElement'
 import Svg from '../Svg'
 import Textarea from '../Textarea'
 
+const message = (id) => {
+  const noteId = `${id}.note`
+  let note = i18n.m(noteId)
+  if (Object.prototype.toString.call(note) === '[object String]' && note.indexOf(noteId) > -1) {
+    note = ''
+  } else {
+    note = <em>{note}</em>
+  }
+
+  return (
+    <div key={newGuid()}>
```

Comment:
Good catch. I'll remove it and see if any warnings are reported.

Reply:
![image](https://user-images.githubusercontent.com/697855/30916674-f96c0c08-a367-11e7-81a2-7e03fb3a20e3.png)

Due to [line 221](https://github.com/18F/e-QIP-prototype/pull/95/files/82021b8009f0fa86c648bc2ebbf3aa80692eb2e1#diff-5a03cfe0364d60af1fa6e31463ddde5bR221)

Reply:
Ah-ha, I see.  Okay, on closer look it makes sense.  Normally you'd put the `key` at the point of iteration, but since the elements are being created inside that `message` function call on 221, you can't put a key on them.  👍   My solution to this in the past has been to wrap the whole thing in a `div`, but this works just as well.  It requires a little bit more contextual understanding of what's happening, but I can't imagine someone familiar with React struggling with it.  😛